support/routes/v2/isobot/ops/botService.ts

```diff
@@ -40,21 +21,13 @@ class BotService {
   }
 
   private checkCname(domain: string) {
-    return ResultAsync.fromPromise(
-      Promise.race([
-        dns.resolveCname(domain),
-        new Promise<null>((_, reject) =>
-          setTimeout(() => reject(null), DNS_TIMEOUT)
-        ),
-      ]),
-      () => {
-        logger.info({
-          message: "Error resolving CNAME",
-          meta: { domain, method: "checkCname" },
-        })
-        return new Error()
-      }
-    )
```

Comment:
we don't need to `Promise.race` because we `await ack()` in the caller - that's the ack within 3s that slack is waiting for.

see [here](https://slack.dev/bolt-js/concepts#commands) under listening and responding to commands